packages/ounit/ounit.2.2.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "Sylvain Le Gall <sylvaini+ocaml@le-gall.net>"
+authors: [ "Maas-Maarten Zeeman" "Sylvain Le Gall" ]
+homepage: "https://github.com/gildor478/ounit"
+dev-repo: "git+https://github.com/gildor478/ounit.git"
+bug-reports: "https://github.com/gildor478/ounit/issues"
+doc: "https://gildor478.github.io/ounit/"
+depends: [
+  "ocamlfind"
```

Comment:
```suggestion
  "ocamlfind" {build}
```
There is no need for ocamlfind to be a runtime dependency, there is no dependency towards `Findlib` and `ounit` doesn't require to be reinstalled every time ocamlfind is upgraded